src/Apps/Collect/routes.tsx

```diff
@@ -54,4 +54,37 @@ export const routes: RouteConfig[] = [
       return { ...initialFilterState, ...params }
     },
   },
+  {
+    // TODO: wont work yet until collection graphql server is ready
+    path: "/collection/:collection_id",
+    Component: CollectApp,
+    // query: graphql`
+    //   query routes_CollectionAppQuery($collectionId: String!) {
+    //     collection(id: $collectionId) {
```

Comment:
Just for my own curiosity, I may have missed this somewhere, but what is `collection` and what's the plan for this? Are 'collection pages' here just scoped to a 'collection' of medium/genes? ie- `/collection/abstract-art`, `/collection/animals` kinda thing?